help/include/sidebar_index.md

```diff
@@ -147,7 +147,7 @@
 * [Pin a channel](/help/pin-a-channel)
 * [Change the color of a channel](/help/change-the-color-of-a-channel)
 * [Unsubscribe from a channel](/help/unsubscribe-from-a-channel)
-* [Manage inactive channels](/help/manage-inactive-channels)
+* [Hide or reveal inactive channels](/help/manage-inactive-channels)
```

Comment:
This does not need any changes to astro.config.mjs since we have also changed the title of `manage-inactive-channels.md` which astro uses as the label for the sidebar entry.